package.json

```diff
@@ -34,7 +34,7 @@
     "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
     "test:ampValidate:ci": "npm run build -- --hide-modules --colors && run-p --race start amp:validate",
     "test:prod:ci": "npm run start & npm install --no-save cypress@latest bbc-a11y@latest lighthouse@latest npm-run-all@latest puppeteer@latest && run-p cypress bbcA11y lighthouse test:puppeteer",
-    "test:prod:jenkins": "npm install --no-save cypress@3.3.0 npm-run-all@4.1.5 && run-p --race start cypress",
+    "test:prod:jenkins": "npm install --no-save cypress@latest bbc-a11y@latest npm-run-all@4.1.5 puppeteer@latest && run-p --race start cypress bbcA11y test:puppeteer",
```

Comment:
Why is any of this script custom? we should just be using `test:prod:ci`

Reply:
We're doing this due to the error below, with which `lighthouse` fails on Jenkins.
`Error: Protocol error (Tracing.start): 'Tracing.start' wasn't found`

The error is due to `Tracing ` being disabled on chromium via Debian packaging as captured by
https://github.com/GoogleChrome/lighthouse/issues/7246
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922431
https://bugs.chromium.org/p/chromium/issues/detail?id=931143

It's reported to be working in `chromium version 72.0.3626.53`. I'll attempt running lighthouse by targeting the specific chromium version from the docker image we use.
The challenge I foresaw with this was locking the chromium version on a shared image hence my making the script custom.